central/policy/datastore/datastore_impl.go

```diff
@@ -20,11 +21,13 @@ import (
 	searchPkg "github.com/stackrox/rox/pkg/search"
 	"github.com/stackrox/rox/pkg/set"
 	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/pkg/uuid"
 )
 
 var (
-	log       = logging.LoggerForModule()
-	policySAC = sac.ForResource(resources.Policy)
+	log          = logging.LoggerForModule()
+	policySAC    = sac.ForResource(resources.Policy)
+	allAccessCtx = sac.WithAllAccess(context.Background())
```

Comment:
Oh, I didn't mean to change the values you were using, just the name of the context

Reply:
ok in that case I prefer keeping name as policyCtx since it is not an all access ctx for sure :)